template.tpl

```diff
@@ -34,14 +34,17 @@ ___VENDOR_DETAILS___
De366YyoVV13UATl4eWuKfUoyAYHio3/V9oD08SuVk69t+9/2v+1/2/+2/2ex/0MWqdkT5R4AJrm4Jt5/8sODTNUB5YqZwHiq0CoYelRloVKGoZKn7+nQPKFwTqVyyJsE9A4vc966ysHe0YjcViGBNFjduUFTw1MlcEUKFZ5uWBSO9jxo1b4SBj28woI9iFz3ebBgZJAeupJG5ydRWzMwXLEU5Wq9xGUBHUi0NC0GZzRIInjriQjKwGEZnnQI8+Y3pMZcwDDQeMdlfCqMwUEWBsap9nFxpLQ65Se3HM5BnmPtQDIe4vLZ+z0cNIp3YZJwm6wS314f2vZP2/6RwbZ/K2vb/7b/h9u/Pw8IiALNCTpWJUxGQKUjCYSWOTNXVEhhGOzdmDWBNbtYRvyOJQ6Xdft2JiTfd+GIxpHccS27lTRK7eXkELfXlzTkLJ4uGZRkGaurlSnLXjohoHqT33/HQkjtvAxDH1NtuTy4TK2UOqHpQPlzTuQPd2LvyA5xUofzvsiUxgvGzl+Mq8giURbHwMNJaDBILz/3M3s9GUxA+XIRFusvREnxAjKZmnnu6m4fzFP6CxKX0S3Jda88aw7J/WYOdwi0H8GgLCW/ccdfecLe9r/tf9v/tv9t/6+O/R9+N/VBSR3Iv2mHtv1OQG/AnNM1rbXzOqFzcDoZrVeIzShjISLjoJOUbE9KjHVNrSru2jaUjB0Jvqc+bIvhDCGLb+HhXUkbLQbIbIyMglB0cj2FHp5omJkmQ+YYBKFek25FLPG4HJ2X0gAeAxUYF8gXpk9wFd/k0I5VeA69xKJDSQOC+EAXcX7tPexdKfqUpOEDvtjQCuOzp7D2gcZTpvo0J55OYqAw0w8uPI2m3r+FYBLkVJ9sHZIvmIGDzlNKGLhACu/70OaCEysMSaMe2reDk5QnZvAcWLvl4Wbb/7b/bf/b/pOM2/6fj/0fINj4ah3KGkRCXCZzyUELU7hZ2a0mKhCNypQGF5UhhJTo+KkTo3jDiPHJjsMbTblPRI1FuSjAdmVbfUByP28hi5k4kcWAq5NciKETglGqzGKqG7KIdUocuKxfMujCDMlNBTqckHVYGGlFotuigiRq23nnyvkAaN9x0ME2IE72KNheTJSNU2X3G64TYxUDkL0Wy3BfO7ovNCWUQwcG0wcLlUFEGAdgEjds+E3enwSuWT8xhYHRSeg/BhMbGqUohgn4ax6fCMW1RPaEhLag95mIU3MDFxjImJItUfC2AcGK5akBk8jb/rf9b/uXbf/b/j96+z/ETHF0ZmZ8vADu0VFMtyYeNiI0Dggnrhot2gcUwZYhNDwzrt7B1Ng9L0ep+EesDt47o5NYhTnBVYnO58X6aOAd3nSiLemWCH2U3DeORlTqy9ZwOiB6XQm+CGFZzh8NNw1IVl62GddvmJoZGA7+k5273MzRPm6U3vvjt41XJosIJf5FosSXhMBYcJCR1MSiGw162zBRmpiMO0NX8k4EWoO+7fJV/bN5FVwPQllGjoEV+qPKbY9DOommgc31LfiEyDQZWgjsZaq1MVxT0+HJrrwrgqlRefqKFtn2v+2ftv1v+9/2T8/f/o/oLKXSoTs3HNAWYSeNRrQ0Eh9ml5CQvDo5PUVFJbFyZYMFiQ4WbqgoSMjmqJmzclNHJrcLy+0DkpXdZRqJOJS9JjOisFlOnZ5TQqtnbi78Gt2Pvd6ExivRncNAonsb2q5PLtjOoFaa6meNF8WUwYVmBYDMAmWv1GY2FrJX88YyQQgmME6CfgrKpiIzXvCOGfWxgWQMYN4LsD+JDaSzhKB6hsFCiHCC8iduGCzTaOR1kWSXDDscD3Yw6CRFZXnEzEko9f9ww2/73/Yfl7b9b/vf9v987P8wi7e1AOgWzhJUDqAA08tklHPl4z50Kmb7Z/2FnZ1es85J2tGqiw7rMLJx6qAEAxPWIW4zp/q4zDwpLeoiubowcka8liQ9xXJNG/LrP7ERzzqVQH38qVWyCIanhXJnJew0IL8PZlxkMEMu/WA2sJKNfNDxhzK/roPVSMupnYznsH0p5fHcbxZ1MyHqgAXjR/RrWbUjqrMWGN1awy3hm/d+iaLZAAUDFTx9+aBmg8aVpXp/gMEBgzpWD/htMmL+mNymUdfbZtv/tn+XYds/1dpt+6dt//Rs9m/7YIkJc8pJE/FFR+dF4VBZWrUVdgSroD4xUJzBJIoa21dWllGt1aGk2uEC0UrIhwa5amj89Lrl+2JBZMH7XPgUI7b+QB39iicTyp1D8+gn693Seppd6zUZgrYw5mQ3Miiv2NlkKIbSrwy/8Fjw804hKAux9xMbfyKT5mNMT4s2YVnLwslYfAABQ1n1l8SaLupm/VAHRJ5sgUE8NPiwjaRnyrLgwI+hiTBySf1pXX/4KeaxgPwgJvOqL2z73/ZP2/5Xddv2v+2fnt3+7SxCh34aULd0nnlpbtXYBSrb/he6RMRW6SEno/IEq2c8T9M3R4zcjJWuUDepQTCv5Wf4p3zGGIOK/P/bu3YlhmEQBv3/b456VxsQBLaOdOkltkE8JJ+TIepTBWGvFvHcw4Dj99VSzch41G3682OfHwU+MGL8HRPipOhrrOHO6ogT6OF2N8f85mWpkTVvIUEYyT4gBX/Bmp1MzXRFNc7JBzb962Czi20ShtMfOUPgfMdOqNfv0XoUrL7rsdKKvZposWjh5fBpDxtPKSZxfrhOMWH5f+0s/5f/y//l/7/4/4lA0AXvHeMhUDAgx+/YDhmeMjYUilOU2u76TN8xeKjg0wtRA26PYW8hw7Ql0RktPi8/ekVqErNn4FrRMPtlTJt7kv1p9ZfiKdfKNkuTo1nDcY72S31tPqcKdR2LqeMgsajx+HUhskROSRPSKFo7YYT7MU7Ll/GQ6k5e4sHkqj87ZSUF1jdI843oUhYi2oj8JAb0zPDN8fYdcW5SlT7QKwzGg7ef5f/yX5b/y//l/x/5/wWAoaMIsultjwAAAABJRU5ErkJggg\u003d\u003d"
```

Comment:
Big-ish change here.
We removed 1 preview image, and replaced it with 4 others.